cmd/conduit/root/pipelines/describe.go

```diff
@@ -169,8 +171,8 @@ func printProcessors(b *strings.Builder, ids []string, indent int) {
 		return
 	}
 
-	fmt.Fprintf(b, "%sProcessors:\n", indentation(indent))
+	fmt.Fprintf(b, "%sProcessors:\n", internal.Indentation(indent))
 	for _, id := range ids {
-		fmt.Fprintf(b, "%s- %s\n", indentation(indent+1), id)
+		fmt.Fprintf(b, "%s- %s\n", internal.Indentation(indent+1), id)
 	}
```

Comment:
Here we only print IDs, while in the connectors describe we print the whole processor config and condition. Would be good to make this the same IMO.

Reply:
@lovromazgon good point